From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Sat, 13 Sep 2026 15:10:00 +0200
Subject: [PATCH] sysbus: grant bootd the ACGs its own clients need

src/bootd/service/ contains four LS2 clients, and the perm file covers one
of them. The other three are refused by the callee with
"Denied method call":

  ActivityManager.cpp:37,61  com.webos.service.activitymanager/complete,
                             /create        -> activity.operation
  Configd.cpp:37             com.webos.service.config/getConfigs
                                            -> systemconfig.query
  SettingsService.cpp:41     com.webos.settingsservice/getSystemSettings
                                            -> settings.query

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending
---
 files/sysbus/com.webos.bootd.perm.json         | 5 ++++-
 1 file changed, 4 insertion(s), 1 deletion(s)

--- a/files/sysbus/com.webos.bootd.perm.json
+++ b/files/sysbus/com.webos.bootd.perm.json
@@ -1,6 +1,9 @@
 {
     "com.webos.bootManager": [
         "application.operation",
-        "application.launcher"
+        "application.launcher",
+        "activity.operation",
+        "systemconfig.query",
+        "settings.query"
     ]
 }
